documentation/proposals/Proposal - Rumble.md

```diff
@@ -0,0 +1,45 @@
+# Summary
+
+Proposal API for gamepad vibration/rumble control. 
+
+# Contributors
+- Dylan P, Ultz Limited
+
+# Current Status
+- [x] Proposed
+- [ ] Discussed with API Review Board (ARB)
+- [ ] Approved
+- [ ] Implemented
+
+# Design Decisions
+
+# Proposed API
+## Interfaces
+### IRumble
+```cs
+public interface IMotor
+{
+    /// <summary>
+    /// The index of this vibration motor.
+    /// </summary>
+    int Index { get; }
+    /// <summary>
+    /// How fast/intense this motor is moving/vibrating, between 0f and 1f.
```

Comment:
I dont like all the slashes, i think we should stick to one word, ie. `/// How intensive this motor is vibrating, between 0f and 1f.
`

Reply:
Fixed.